engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/process/FlinkSource.scala

```diff
@@ -18,7 +20,11 @@ import scala.reflect._
 trait FlinkSource[T] extends Source[T] {
```

Comment:
Do we really need T here anymore? Maybe for Flink sources we can just have 
```FlinkSource extends Source[Context]``` (or maybe... in the future we can remove ```T``` from source altogether?

Reply:
I don't like `Source[Context]` approach, because until now `T` was representing single type added to context by source. IMO would be better to remove it at all or at least left here TODO for it.

Reply:
True that... Let's add TODO to refactor Source trait then. We have to somehow make "simple" case - one input variable, - easy as it now, while also enabling more complex cases. But for now TODO will be ok, as this change is already v. big :)